RELEASES.md

```diff
@@ -25,7 +25,7 @@ Compiler
 --------
 - [Rustc now respects the `-C codegen-units` flag in incremental mode.][70156]
   Additionally when in incremental mode rustc defaults to 256 codegen units.
-- [Refactored `catch_unwind`, to have zero-cost unless unwinding is enabled and
+- [Refactored `catch_unwind` to have zero-cost, unless unwinding is enabled AND
```

Comment:
I don't think the capitalization of AND is necessary? But moving the comma makes sense to me.